gbif/coordinator/tasks/src/main/java/org/gbif/pipelines/tasks/PipelinesCallback.java

```diff
@@ -428,12 +428,15 @@ private void updateQueuedStatus(TrackingInfo info) {
     for (Graph<StepType>.Edge e : nodeEdges) {
       PipelineStep step = info.pipelineStepMap.get(e.getNode());
       if (step != null && !PROCESSED_STATE_SET.contains(step.getState())) {
-        // Call Registry to change the state to queued
-        log.info("History client: set pipeline step to QUEUED: {}", step);
-        Retry.decorateRunnable(
-                RETRY, () -> historyClient.setSubmittedPipelineStepToQueued(step.getKey()))
-            .run();
-        log.info("Step {} with step key {} as QUEUED", step.getType(), step.getKey());
+        step.setState(PipelineStep.Status.QUEUED);
```

Comment:
it's the other way around. The deleted block should be restored and the other deleted